agent-bridge/src/main/java/com/newrelic/agent/bridge/ExtensionHolder.java

```diff
@@ -17,7 +18,7 @@
      * Return the value associated with the instance in this ExtensionHolder. If the value is not present it will be
      * initialized (in a thread-safe manner) using valueLoader.
      */
-    public T getExtension(Object instance, Callable<T> valueLoader);
+    public T getExtension(Object instance, Supplier<T> valueLoader);
```

Comment:
I found only one implementation of this interface. If it _is_ used elsewhere we should revert this change and introduce a `try/catch` construct in the implementation.